<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>db_window</class>
 <widget class="QMainWindow" name="db_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="add">
        <property name="text">
         <string>Add subjects...</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/icons/open.xpm</normaloff>:/icons/icons/open.xpm</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_subject">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/icons/delete.xpm</normaloff>:/icons/icons/delete.xpm</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="move_down">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/icons/down.xpm</normaloff>:/icons/icons/down.xpm</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="move_up">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/icons/up.xpm</normaloff>:/icons/icons/up.xpm</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Fiber threshold (otsu)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="fp_coverage">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.600000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="normalize_fp">
        <property name="text">
         <string>Normalize FP</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QTableWidget" name="subject_list">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>4</verstretch>
               </sizepolicy>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
             <widget class="QTextBrowser" name="report">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QTabWidget" name="subject_view">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Values</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QGraphicsView" name="fp_view">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>100</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGraphicsView" name="vbc_view"/>
               </item>
               <item>
                <widget class="QScrollBar" name="slice_pos">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_20">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_14">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QRadioButton" name="view_x">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../icons.qrc">
                       <normaloff>:/icons/icons/sag.xpm</normaloff>:/icons/icons/sag.xpm</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="x_pos"/>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="view_y">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../icons.qrc">
                       <normaloff>:/icons/icons/coronal.xpm</normaloff>:/icons/icons/coronal.xpm</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="y_pos"/>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="view_z">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../icons.qrc">
                       <normaloff>:/icons/icons/axial.xpm</normaloff>:/icons/icons/axial.xpm</iconset>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="z_pos"/>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_13">
                     <property name="text">
                      <string>Zoom</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="zoom">
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="decimals">
                      <number>1</number>
                     </property>
                     <property name="minimum">
                      <double>0.100000000000000</double>
                     </property>
                     <property name="maximum">
                      <double>20.000000000000000</double>
                     </property>
                     <property name="value">
                      <double>4.000000000000000</double>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="show_mask">
                     <property name="text">
                      <string>show mask</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>70</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_19">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Difference Matrix</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGraphicsView" name="fp_dif_view">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_23">
                  <item>
                   <widget class="QLabel" name="label_15">
                    <property name="text">
                     <string>Zoom</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="fp_zoom">
                    <property name="minimumSize">
                     <size>
                      <width>50</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="decimals">
                     <number>1</number>
                    </property>
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>10.000000000000000</double>
                    </property>
                    <property name="value">
                     <double>3.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="calculate_dif">
                    <property name="text">
                     <string>Re-calculate</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File&amp;</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionCurrent_Subject"/>
     <addaction name="actionAll_Subjects"/>
    </widget>
    <addaction name="actionAdd_DB"/>
    <addaction name="actionSave_DB_as"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Subject_Name_as"/>
    <addaction name="action_Save_R2_values_as"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Demographics"/>
    <addaction name="actionSave_Demographics"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCalculate_change"/>
    <addaction name="actionSelect_Subjects"/>
   </widget>
   <widget class="QMenu" name="menuFingerprinting">
    <property name="title">
     <string>Fingerprinting</string>
    </property>
    <addaction name="actionSave_fingerprints"/>
    <addaction name="actionSave_pair_wise_difference_as"/>
    <addaction name="separator"/>
    <addaction name="actionSave_mask"/>
    <addaction name="actionLoad_mask"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFingerprinting"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_DB_as">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save DB as...</string>
   </property>
  </action>
  <action name="actionCalculate_change">
   <property name="text">
    <string>Logitudinal scans...</string>
   </property>
  </action>
  <action name="actionSave_Subject_Name_as">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save Subject Name as...</string>
   </property>
  </action>
  <action name="action_Save_R2_values_as">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save R2 values as...</string>
   </property>
  </action>
  <action name="actionSave_fingerprints">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save fingerprints...</string>
   </property>
  </action>
  <action name="actionSave_pair_wise_difference_as">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save pair-wise difference as...</string>
   </property>
  </action>
  <action name="actionAdd_DB">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/open.xpm</normaloff>:/icons/icons/open.xpm</iconset>
   </property>
   <property name="text">
    <string>Add Subjects...</string>
   </property>
  </action>
  <action name="actionLoad_mask">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/open.xpm</normaloff>:/icons/icons/open.xpm</iconset>
   </property>
   <property name="text">
    <string>Load fingerprint mask...</string>
   </property>
  </action>
  <action name="actionSave_mask">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save fingerprint mask...</string>
   </property>
  </action>
  <action name="actionSelect_Subjects">
   <property name="text">
    <string>Select Subjects...</string>
   </property>
  </action>
  <action name="actionCurrent_Subject">
   <property name="text">
    <string>Current Subject</string>
   </property>
  </action>
  <action name="actionAll_Subjects">
   <property name="text">
    <string>All Subjects...</string>
   </property>
  </action>
  <action name="actionAdd_Demographics">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/open.xpm</normaloff>:/icons/icons/open.xpm</iconset>
   </property>
   <property name="text">
    <string>Add Demographics...</string>
   </property>
  </action>
  <action name="actionSave_Demographics">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save Demographics...</string>
   </property>
  </action>
  <action name="actionOpen_Demographics">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/open.xpm</normaloff>:/icons/icons/open.xpm</iconset>
   </property>
   <property name="text">
    <string>Open Demographics...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
